### PR TITLE
Add original repo files

### PR DESCRIPTION
**Pull request by mullenlucas**
Add original files of the project made during trial session, and make reviews according to learned skills on the first week at microverse

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- <h1>Hello World!, we are software developers!</h1> -->
+    <script  src="js/index.js"></script>
+    <div id="title-header" style="margin-left: 350px;
+    font-size: 20px; padding: 25px; color: #E900FF;">
+      <h1>JavaScript reference</h1>
+    </div>
+    <nav id="navbar">
+      <header id="nav-header">
+        <h2>JS documentation</h2>
```
Comment (mullenlucas) [inline index.html:21]: I'd suggest that the header should contain the nav-bar, and not the other way around

File: index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    </nav>
+    <main id="main-doc">
+      <section class="main-section" id="Built_ins">
+        <header class="subtitle">
+          <h2>Built-ins</h2>
+        </header>
+
+      </section>
+      <section class="main-section" id="Value_properties">
+        <header class="subtitle">
```
Comment (mullenlucas) [inline index.html:37]: I'd suggest using only one `<header>` tag at the beginning of the document for the top bar, and use an `<h2>` tag on all this "subtitle" classes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+          <h2>Value properties</h2>
+        </header>
+        <ul>
+          <li><a href="#Value_properties">globalThis</a></li>
```
Comment (mullenlucas) [inline index.html:41]: I'd try using indentations on these sections so as to keep the code more organized and easier to review

File: index.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+  <code>
+let re = /ab+c/i; // literal notation
+let re = new RegExp('ab+c', 'i') // constructor with string pattern as first argument
+let re = new RegExp(/ab+c/, 'i') // constructor with regular expression literal as first argument (Starting with ECMAScript 6)
+</code></pre>
+        </ul>
+      </section>
+
+    </main>
+<div id="footer">
```
Comment (mullenlucas) [inline index.html:211]: A better practice than setting a footer div with the id="footer", is to use the `<footer>` tag directly for the footer section of the html

**Review by mullenlucas — COMMENTED** [review]
The structure of the code appears fairly well. Some tags could've been used differently and the code organization aswell. Also, it is a fairly simple website and no flexbox or grid has been used, although it could've been included

**Review by algerina — APPROVED** [review]
Hi @mullenlucas , 
Good job so far!

May I highlight the following:

- You have a good review with inline comments, provided politly PR:heavy_check_mark:


## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**